perllib/FixMyStreet/App/Controller/Admin/Triage.pm

```diff
@@ -0,0 +1,108 @@
+package FixMyStreet::App::Controller::Admin::Triage;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'FixMyStreet::App::Controller::Reports'; }
+
+sub auto : Private {
```

Comment:
The auto in Admin.pm should also be being run, so only the triage bit of this function should be needed?